swift/apple/FirezoneKit/Sources/FirezoneKit/Helpers/DeviceMetadata.swift

```diff
@@ -22,21 +22,24 @@ public class DeviceMetadata {
     return !fileExists
   }
 
-  public static func getDeviceName() -> String? {
+  public static func getDeviceName() -> String {
     // Returns a generic device name on iOS 16 and higher
     // See https://github.com/firezone/firezone/issues/3034
-    #if os(iOS)
-      return UIDevice.current.name
-    #else
-      // Fallback to connlib's gethostname()
```

Comment:
Just yesterday I looked into whether we can get rid of connlib's `hostname` function! Great to see that we have one less place now where it is relied on. IMO how we currently build this header is pretty convoluted with all these `Option`s.